product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx

```diff
@@ -1,12 +1,19 @@
 ---
 title: "Setting up your identity provider"
 navTitle: "Setting up your identity provider"
-description: Applies when you purchased BigAnimal directly from EDB
+description: How to set up your identify provider when you purchased BigAnimal directly from EDB
```

Comment:
Does something more general like this work:

"How to set up your identity provider **with an EDB account**."

Reply:
I think it is just "How to set up your identity provider"? 

The "with an EDB account" gives me a feeling of "use your EDB Account to set up your identity provider" which is a bit confusing. (because *account != identity provider*).